frontend/src/views/ManagementPage/CanteensPagination.vue

```diff
@@ -136,14 +159,22 @@ export default {
       if (replace) this.$router.replace(page).catch(() => {})
       else this.$router.push(page).catch(() => {})
     },
+    filterProductionType() {
+      this.populateProductionType()
+      this.page = 1
+      this.$router.push({ query: this.query }).catch(() => {})
+    },
     $route() {
       this.populateInitialParameters()
       this.fetchCurrentPage()
```

Comment:
Not sure production type is fully updated on route change - when I navigate back and forward in browser the filter value is updated but not the canteen list

Reply:
Thanks for spotting this. Fix : https://github.com/betagouv/ma-cantine/pull/2038/commits/0c4fa1e25f0c806f5877f79d60f819c974b9e8bf